src/utils/validate-object-id-hook.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const errors = require('feathers-errors');
+const mongoose = require('mongoose');
+
+var validateObjectId = function() {
+  return function(hook) {
+    const id = hook.id;
+    if (id && !mongoose.Types.ObjectId.isValid(id)) {
```

Comment:
I think I commented in the wrong following PR. (I should comment here).

So, can't model handle this already?